demisto_sdk/scripts/update_additional_dependencies.py

```diff
@@ -24,7 +24,7 @@ def update_additional_dependencies(
         int: 1 if failed, 0 if succeeded
     """
     try:
-        if "poetry.lock" not in GitUtil()._get_all_changed_files():
+        if Path("poetry.lock") not in GitUtil()._get_all_changed_files():
```

Comment:
Is this safe, considering relative paths?

Reply:
This function  _get_all_changed_files returns the relative paths that was changed

Reply:
relative to the git path, but it's the same

Reply:
Perhaps `CONTENT_PATH / "poetry.lock"` would be safer

Reply:
but it will not work